client/scss/layouts/_account.scss

```diff
@@ -1,29 +1,23 @@
 .avatar-panel {
-  margin-top: 20px;
-
   &__image {
-    float: left;
-    width: 16%;
-    margin-bottom: 20px;
+    .avatar-clear:checked ~ .avatar.avatar-span {
```

Comment:
Because this CSS is a bit confusing let's add a comment above their block to explain